src/ext.rs

```diff
@@ -1149,6 +1217,16 @@ fn to_ipv6mr_interface(value: u32) -> c_uint {
     value as c_uint
 }
 
+#[cfg(target_os = "android")]
+fn from_ipv6mr_interface(value: c_int) -> u32 {
```

Comment:
I just noticed other areas of this code base just do direct casting, I'll remove this.